web/crypto_utils.py

```
@@ -0,0 +1,149 @@
+            return b""
+
+        def decrypt(self, token):
+            return b""
+
+
+from django.conf import settings
+from django.db import models
+
+
+# Generate a fixed key from the SECRET_KEY to ensure consistent encryption/decryption
+def get_encryption_key():
```

Ruff (0.8.2)

Missing return type annotation for public function `get_encryption_key`

(ANN201)